src/main/java/Yoj.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Scanner;
+//so scanner class can be used
+
+public class Yoj {
+    public static void printLine() {
+        System.out.println("________________________________________");
+    }
+    public static void printShortLine() {
+        System.out.println("_____________");
+    }
+    public static void printHello() {
+        String logo =
+                "__   __   ___    _____ \n"
+                        + "\\ \\ / /  / _ \\  | ___ |\n"
+                        + " \\ Y /  | | | |     | | \n"
+                        + "  \\ /   | | | |     | | \n"
+                        + "  | |   | |_| |  ___| | \n"
+                        + "  |_|    \\___/  |____/          \n";
+        printLine();
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm YOJ");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    public static void main(String[] args) {
+        printHello();
+        // get user input
+        Scanner in = new Scanner(System.in);
+        String userInput;
+        userInput = in.nextLine();
+
+        while(!userInput.equals("bye")) {
+            printShortLine();
+            System.out.println("added: " + userInput);
+            printLine();
+            if (userInput.equals("list")) {
+                Task.printList();
+            } else if(userInput.matches("mark \\d+")) {
```

Comment:
You may want to declare all the regex expressions being used as constants. This also allows you to just update the constant and reuse that at multiple places, without needing to update it at every place you declared it initially.